src/box/lua/config/configdata.lua

```diff
@@ -65,6 +66,116 @@ function methods.names(self)
     }
 end
 
+local function new_uuid(str)
+    local sha = digest.sha1_hex(str)
+    return sha:sub(1,8)..'-'..sha:sub(9,12)..'-'..sha:sub(13,16)..'-'..
+           '00'..sha:sub(17,18)..'-'..sha:sub(19,30)
+end
```

Comment:
I found the name a bit confusing. It is not just some new uuid. It is a kind of a hash function and it is fully determined by the given source string.

How about something like `sha1_as_uuid()`? Or, maybe, `uuid_from_name()`?

Reply:
Renamed to `uuid_from_name()`.